### PR TITLE
Add pagination to scheduled tasks

### PR DESCRIPTION
**Pull request by crobibero**
This change may fix slow db queries from normal Jellyfin browsing

Untested.

### CODE REVIEW

File: Trakt/ScheduledTasks/SyncLibraryTask.cs [hunk 1]
```diff
@@ -127,81 +127,94 @@ public async Task ExecuteAsync(IProgress<double> progress, CancellationToken can
...
+                var userData = _userDataManager.GetUserData(user.Id, libraryMovie);
+
+                if (traktUser.SynchronizeCollections)
                 {
-                    collectedMovies.Add(libraryMovie);
+                    // if movie is not collected, or (export media info setting is enabled and every collected matching movie has different metadata), collect it
+                    var collectedMathingMovies = SyncFromTraktTask.FindMatches(libraryMovie, traktCollectedMovies).ToList();
+                    if (!collectedMathingMovies.Any()
+                        || (traktUser.ExportMediaInfo
+                            && collectedMathingMovies.All(
```
Comment (Shadowghost) [inline Trakt/ScheduledTasks/SyncLibraryTask.cs:171]: I guess this should be `collectedMatchingMovies`?

Comment (dannymichel) [reply Trakt/ScheduledTasks/SyncLibraryTask.cs:171]: Should i make this change in my fork as well?

Comment (crobibero) [reply Trakt/ScheduledTasks/SyncLibraryTask.cs:171]: Likely, but I didn't change that :)

Comment (crobibero) [reply Trakt/ScheduledTasks/SyncLibraryTask.cs:171]: It's a local variable, so not needed. I'll fix the typo after I finish testing

**Pull request merged**